implementation/src/main/java/io/smallrye/health/checks/HealthChecksLogging.java

```diff
@@ -0,0 +1,25 @@
+package io.smallrye.health.checks;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+@MessageLogger(projectCode = "SRHCK", length = 5)
+interface HealthChecksLogging extends BasicLogger {
```

Comment:
I am not that much familiar with JBoss logging, but can't we have only single logger class with all messages? Or is that an antipattern?

Reply:
Me neither. Just follow the procedure [here](https://github.com/smallrye/smallrye-parent/wiki/I18N-Messages-and-Logging#general-requirements). 

If you feel that is something should be changed, tell me and I will do it.